aspnetcore/blazor/javascript-interop.md

```diff
@@ -297,11 +297,11 @@ Console output in the browser's web developer tools:
 Hello, Blazor!
 ```
 
-## Share interop code in a Razor Component class library
+## Share interop code in a Razor component class library
```

Comment:
Replace "Razor Component class library" with "Blazor class library" in this section (`dotnet new blazorlib`). We don't have support for embedding static assets in Razor Class Libraries yet.